ee/control/control.go

```diff
@@ -110,6 +112,45 @@ func (cs *ControlService) Stop() {
 	}
 }
 
+func (cs *ControlService) AccelerateRequestInterval(interval, duration time.Duration) error {
+	// preform a fetch now
+	if err := cs.Fetch(); err != nil {
+		// if we got an error, log it and move on
+		level.Error(cs.logger).Log("msg", "failed to fetch data from control server. Not fatal, moving on", "err", err)
+	}
+
+	if interval <= 0 || duration <= 0 {
```

Comment:
I think we should define a minimum interval, I'd say 5 or 10s, and use that if it's lower

Reply:
done